actors.md

```diff
@@ -708,3 +708,233 @@ func UpdatePeerID(pid PeerID) {
 ## Payment Channel Broker Actor
 
 TODO
+
+## Multisig Account Actor
+
+A basic multisig account actor. Allows sending of messages like a normal account actor, but with the requirement of M of N parties agreeing to the operation. Any party can veto any operation (which is a potentially debateable design decision). Completed and/or cancelled operations stick around in the actors state until explicitly cleared out.
+
+#### State
+
+```go
+type Multisig struct {
+    Signers []Address
+    Required uint
+
+    NextTxID uint64
+    Transactions map[int]Transaction
+}
+
+type Transaction struct {
+    TxID uint64
+    To Address
+    Value TokenAmount
+    Method string
+    Params []byte
+    Approved []Address
+    Completed bool
+    Canceled bool
+}
+```
+
+
+
+#### Constructor
+
+>  This method sets up the initial state for the multisig account
+
+#### Parameters
+
+| Name     | Type       | Description                                         |
+| -------- | ---------- | ------------------------------------------------------------ |
+| `signers`   | `[]Address`      | The addresses that will be the signatories of this wallet. |
+| `required` | `uint` | The number of signatories required to perform a transaction.       |
+
+```go
+func Multisig(signers []Address, required uint) {
+    self.Signers = signers
+    self.Required = required
+}
+```
+
+
+
+### Propose
+
+> Propose is used to propose a new transaction to be sent by this multisig. The proposer must be a signer, and the proposal also serves as implicit approval from the proposer. If only a single signature is required, then the transaction is executed immediately.
+
+| Name     | Type       | Description  |
+| --- | --- | --- |
+| `to` |  `Address` | The address of the target of the proposed transaction. |
+|  `value` | `TokenAmount` | The amount of funds to send with the proposed transaction |
+|  `method` | `string` | The method that will be invoked on the proposed transactions target. |
+| `params` | `[]byte` | The parameters that will be passed to the method invocation on the proposed transactions target. |
+
+```go
+func Propose(to Address, value TokenAmount, method string, params []byte) uint64 {
+    if !isSigner(msg.From) {
+        Fatal("not authorized")
+    }
```

Comment:
Shouldn't you conceivably be allowed to propose a transaction for which you are not a signer?

Reply:
I don't think thats useful. Then any random person can spam your multisig with bad transactions that we have to clear out, and can be misleading (making a typo on an approval transaction much more risky)